Shawn/Shawn_BuyMeALaptop_Challenge.py

```diff
@@ -0,0 +1,117 @@
+        },
+        {
+            "id": "2",
+            "productName": "Dell XPS",
+            "url": "https://www.dell.com/en-us/shop/dell-laptops/sr/laptops/xps-laptops?~ck=bt",
+            "types": [
+                {
+                    "id": "3",
+                    "screen_size": "13-inch",
+                    "cpu": ["11th Generation Intel Core i3-1115G4 Processor", "11th Generation Intel Core i5-1135G7 Processor"],
+                    "ram": ["8GB"],
+                    "storage": ["256GB SSD","512 GB SSD", "1 TB SSD"],
+                    "colors": ["Platinum silver exterior, black interior", "Platinum silver exterior, black interior"],
+                    "price": [999, 1099, 1149, 1249]
+                },
+                {
+                    "id": "4",
+                    "screen_size": "15-inch",
+                    "cpu": ["10th Generation Intel Core i5-10300H"],
+                    "ram": ["8GB", "16GB", "32GB", "64GB"],
+                    "storage": ["256 GB SSD", "512 GB SSD", "1 TB SSD", "2 TB SSD"],
+                    "colors": ["Frost White with White Palmrest", "Frost White with White Palmrest"],
+                    "price": [1199, 1299, 1399, 1749, 1999, 2299]
+                }
+            ],
+            "description": "The Dell XPS is an absolutely brilliant laptop. The 2020  version rocks an 11th-generation Intel Core i3, i5 or i7 processor and a bezel-less Infinity Edge display. This Dell XPS continues to be the most popular Windows laptop in the world. What is more, there is a wide range of customization options, so you can really make the Dell XPS the best laptop for your needs. "
+        }
+]
+
+# TODO: Write code to print out the MacBook Pro url 
+print('Apple Macbook Pro url')
+for laptop in laptops:
+    print(laptop['url'])
+print()
+
+# TODO: Write code to print all possible prices of the 16-inch MacBook Pro.
+MacBook_Pro_prices= laptops[0]['types'][1]['price']
+for price in MacBook_Pro_prices:
+    print(price)
+print()
+
+# TODO: Write code to print all the color options for Dell XPS 13.
+color_options= laptops[1]['types'][0]['colors']
+print(color_options)
+print()
+
+# TODO: Write code to print the description of Dell XPS laptop.
+print('The Dell XPS is an absolutely brilliant laptop. The 2020  version rocks an 11th-generation Intel Core i3, i5 or i7 processor and a bezel-less Infinity Edge display. This Dell XPS continues to be the most popular Windows laptop in the world. What is more, there is a wide range of customization options, so you can really make the Dell XPS the best laptop for your needs. ')
```

Comment:
The premise of the challenge is to key into the nested data, do not just print the description in a print statement. Use the same way you did color options